src/views/assets/Asset/AssetList/components/const.js

```diff
@@ -139,6 +139,7 @@ export function getDefaultConfig(vm) {
         sortable: true
       },
       platform: {
+        width: '160px',
         sortable: true,
         formatter: PlatformFormatter
       },
```

Comment:
The main difference between the two lines of code is in the line with the curly braces `{}` instead of `{width}`, which changes the value of `<` "platform.width"`. To optimize the readability and maintainability of the code:

```javascript
export function getDefaultConfig(vm) {

  // Here's the rest of your existing functionality...
}
```

This change might not have an impact on regular operations but could improve style slightly. It's generally advisable to ensure consistency in naming conventions (e.g., `height`, `width`) since it makes files easier to read and understand.

If you want further optimizations, let me know!